draft-twi-reference-architecture.md

```diff
@@ -49,72 +48,110 @@ informative:
+* Insider threats, and other reasons for protecting data-in-use
+
+Correspondingly, there is an increased push to harmonize management and governance of human and non-human identities.
+Modern workloads may operate on their own behalf with their own credentials, or as agents on behalf of other entities with delegated credentials.
+Entities interested in strong assurances around the security of their deployed workloads, for regulatory, contractual or peace of mind reasons, are starting to face large and challenging tasks of upgrading their existing computing system infrastructure to meet these requirements.
+Current ways of issuing and managing workload identities, as well as those required for effective protection of data-in-use, are subject to multiple architectural challenges; chief among them:
 
 1. Lack of workload isolation against the hardware and the operating system owners/administrators, as well as peer workload instances
 2. Lack of strong binding between a workload credential and the workload instance to which that credential had been issued
-3. Inability to associate a credential with a set of decisions leading up to its issuance
-_Note that these requirements are related: lack of process isolation eases credential exfiltration and leads to credential leakage and reuse._
+3. Lack of verifiable composition of the workload, and inability to associate a credential with a set of decisions leading up to its issuance
 
-In the immediate term, effective Confidential Computing faces a significant challenge: the scale of uplift and immature tooling are both clear barriers. Longer term, however, Confidential Computing provides a vital improvement due to its unique features and broad availability. Confidential Computing-assisted mechanisms have to fit inside the emerging Workload Identity solution ecosystem. The evolution of this ecosystem should remain in alignment with the expectations of the owners and operators of Confidential Computing workloads. These efforts will build on the concept of _Trustworthy Workload Identity_ defined below. Data-in-use protection of workloads that have such identities will be a critical downstream effect.
+It is important to highlight that these requirements are related: lack of process isolation eases credential exfiltration and leads to credential leakage and reuse.
 
-While it is quite clear that for the foreseeable future this need is not going to be met by Confidential Computing alone, the security features related to workload identification and isolation offered by Confidential Computing eclipse other approaches. Confidential Computing will not be first to market, and thus will need to evolve to fit within, and ideally serve as the North Star of workload identity as it becomes a foundational pillar of trustworthy and governable enterprise computing.
+In the immediate term, effective Confidential Computing faces a significant challenge: the scale of uplift and immature tooling are both clear barriers.
```

Comment:
As CC is gaining popularity now, shall we delete line 76..? The reason been, it water down a bit our efforts for a need for CC?